### ISSUE TITLE
Staging Review finding: Extra `aria-label` text creates potential barrier for voice command software users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#94524]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Identity
**Product:** Sign in page (and sign in modal)
**Feature:** Sign in Transition

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In both the standalone page and the modal, multiple links and buttons have an `aria-label` that provides a lengthier description of what the element is:
- The Login.gov button has an `aria-label` "Sign in with Login.gov"
- The ID.me button has "Sign in with ID.me"
- The My HealtheVet button has "Sign in with My HealtheVet"
- The DS Logon button has "Sign in with DS Logon"
- The "signing in" link has "Questions about signing in to VA.gov. (Opens a new window)"
- The "verifying your identity" link has "Verifying your identity on VA.gov. (Opens a new window)"

In each of these cases, the `aria-label` doesn't really add anything of value that's not already clear from context (except the "opens in a new window" text, which will be addressed in anther issue). It's well-intentioned but ultimately doesn't seem necessary. (Since the production version of the page also as these, I'm guessing they've been there for a while.)

But unnecessary `aria-label` text introduces a new barrier for voice command software users. The typical interaction model for most sighted voice users would be to speak a command like "Click [text]," where [text] is the visible text on the page: "Click verifying your identity" for example.

Voice command software checks the browser's accessibility tree for matches or partial matches, so when you say "Click Login.gov," it's looking for an element named "Login.gov." But when there's an `aria-label`, that text replaces the visible text in the accessibility tree. Some voice command software is good at finding partial matches (eg. matching "Login.gov" to "Sign in with Login.gov"), and some software really struggles if the text string spoken isn't also the start of the name in the accessibility tree. Apple Voice Control is a particularly bad offender here.

Voice users typically have workarounds available to them, so when there's a compelling reason to add an `aria-label` for screen reader users it's justifiable to introduce this barrier for voice users. But in this case I don't think the `aria-label` adds enough value to justify it.

<details><summary>Link, screenshot or steps to recreate</summary>

Example of an element coded with an unnecessary `aria-label`:

```
<button type="button" aria-label="Sign in with My HealtheVet" class="usa-button mhv-button vads-u-margin-y--1p5 vads-u-padding-y--2" data-csp="mhv" aria-describedby="mhvH3">My HealtheVet</button>
```

</details>

## Recommended action

Recommendations:
- Change the `aria-label` for the Login.gov button to just read "Login.gov"
- Change the `aria-label` for the ID.me button to just read "ID.me"
- Remove the `aria-label` from the My HealtheVet button
- Remove the `aria-label` from the DS Logon button
- Remove the `aria-label` from "signing in" link and change the link text to "questions about signing in (opens in a new tab)" (moving the two words immediately before the link into the link and adding the "opens in a new tab" to the visible text)
- Remove the `aria-label` from the "verifying your identity" link and change the link text to "verifying your identity (opens in a new tab)"

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
